compiler/luci-interpreter/src/kernels/Conv2D.test.cpp

```diff
@@ -70,49 +85,7 @@ TEST(Conv2DTest, Float)
   EXPECT_THAT(extractTensorShape(output_tensor), ::testing::ElementsAreArray(ref_output_shape));
 }
 
-TEST(Conv2DTest, FloatPointwise)
```

Comment:
I this test is lost during rebase

Reply:
Return and add memory manager to this test